modules/databend/databend.go

```diff
@@ -0,0 +1,139 @@
+package databend
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const (
+	databendUser        = "databend"
+	defaultUser         = "databend"
+	defaultPassword     = "databend"
+	defaultDatabaseName = "default"
+)
+
+// DatabendContainer represents the Databend container type used in the module
+type DatabendContainer struct {
+	testcontainers.Container
+	username string
+	password string
+	database string
+}
+
+func WithDefaultCredentials() testcontainers.CustomizeRequestOption {
```

Comment:
One last thing before running the CI: as this function is exposed in the API, I think we need to add it in the docs, unless it's private and not intended to be part of the public API.

Wdyt about that? Do we want users to define default credentials, or we should keep it internal to the module as an implementation detail?

Reply:
Sorry, I have not seen the `WithDefaultCredentials ` function docs in mysql.md,  maybe we should keep it internal to the module.

Reply:
Yeah, each module comes with its own set of configuration options, and its own set of internals. I agree, this option seems internal to the databend module

Reply:
This cant be private as the user needs to call pass it to `Run`

Reply:
@stevenh You are right. IMHO, this method is redundant, I should delete this method.